IM_calculation/IM/im_calculation.py

```diff
@@ -226,7 +226,7 @@ def calc_CAV(waveform, times, im, comps_to_store, comps_to_calculate):
         func = lambda x: intensity_measures.get_cumulative_abs_velocity_nd(
             np.squeeze(x), times=times
         )
-        rotd = calculate_rotd(np.expand_dims(waveform, 0), comps_to_store, func)
+        rotd = calculate_rotd(waveform, comps_to_store, func)
```

Comment:
just being a bit pedantic here. `func=func` will be nice to be consistent with other usages.